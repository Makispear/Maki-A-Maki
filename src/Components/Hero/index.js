import React from "react";

function Hero() {
  return (
    <section id="hero">
      <img src="assets/images/measdfvb.jpg" alt="Picture of Maki smiling with A river in the back" />
      <h1 data-testid="name">Maki Aboabida</h1>
      <h2 data-testid="occupation">Full-Stack Web Developer</h2>
    </section>
  )
}

export default Hero
