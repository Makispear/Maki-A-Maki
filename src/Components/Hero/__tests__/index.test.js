import React from "react";
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Hero from '..';

afterEach(cleanup)

describe('Hero Component', () => {
  it('renders', () => {
    render(<Hero />)
  })

  it('Matches snapshot', () => {
    const { asFragment } = render(<Hero />)

    expect(asFragment).toMatchSnapshot()
  })

  it('Contains ocupation and name', () => {
      const { getByTestId } = render(<Hero />)
  
      expect(getByTestId('name')).toHaveTextContent('Maki Aboabida')
      expect(getByTestId('occupation')).toHaveTextContent('Full-Stack Web Developer')
  })
})
