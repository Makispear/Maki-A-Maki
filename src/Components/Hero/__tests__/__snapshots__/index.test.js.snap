// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Component Matches snapshot 1`] = `[Function]`;
